ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vlan-aware-bundles.j2

```diff
@@ -1,7 +1,9 @@
 {# tenant router bgp evpn configuration - MAC-VRF VLAN Aware bundles #}
   vlan_aware_bundles:
-{% if vxlan_vlan_aware_bundles is arista.avd.defined(true) %}
+{% set evpn_vlan_aware_bundles = vxlan_vlan_aware_bundles | arista.avd.default(evpn_vlan_aware_bundles, false) %}
+{% if evpn_vlan_aware_bundles == true %}
```

Comment:
Please don't compare boolean. I would recommend ```if evpn_vlan_aware_bundles```

Reply:
If you search throughout AVD, we compare boolean everywhere, to avoid just accepting "trueish" test. Otherwise any value would work. (Ex. also the string "false")

Reply:
same when we do `is arista.avd.defined(true)`

Reply:
Agreed, I would apply that boolean logic only to internal variables not to user input.